std/digest/murmurhash.d

```diff
@@ -535,7 +535,14 @@ struct MurmurHash3(uint size /* 32 or 128 */ , uint opt = size_t.sizeof == 8 ? 6
                 const numFullChunks = data.length / TChunk.sizeof;
                 foreach (ref const chunk; cast(const(TChunk)[]) data[0 .. numFullChunks * TChunk.sizeof])
                 {
-                    const block = cast(Element) chunk;
+                    static if (size == 32)
+                    {
+                        import core.stdc.string : memcpy;
+                        Element block;
+                        ()@trusted{memcpy(&block, &chunk, Element.sizeof);}();
+                    }
```

Comment:
Put this in to fix the 32-bit unaligned read you pointed out is still there, @kinke, following David's suggestion.  I'm not having any issues with the way it was before or now, ie the test passes, but this won't screw up with some optimizer pass later on.